Gems/Atom/RHI/DX12/Code/Source/RHI/MemoryView.cpp

```diff
@@ -93,12 +93,22 @@ namespace AZ
         {
             if (m_memoryAllocation.m_memory)
             {
+                // NOTE:  This assumes that name is null-terminated.
                 AZStd::wstring wname;
                 AZStd::to_wstring(wname, name);
                 m_memoryAllocation.m_memory->SetName(wname.data());
             }
         }
 
+        void MemoryView::SetName(const AZStd::wstring_view& name)
+        {
+            if (m_memoryAllocation.m_memory)
+            {
+                // NOTE:  This assumes that name is null-terminated.
```

Comment:
Nitpick: This comment should perhaps also be added to the header so users that just look at the header (indirectly through tools as well) are aware of this. Alternatively use `const wchar*` instead of `AZStd::wstring_view` perhaps.